nucliadb/nucliadb/reader/tests/test_reader_file_download.py

```diff
@@ -206,17 +206,28 @@ async def _get_message_with_file(test_resource):
     return msg_id, file_num
 
 
+FILE_SIZE = 10
+
+
 @pytest.mark.parametrize(
-    "range_request,filesize,start,end,exception",
+    "range_request,filesize,start,end,range_size,exception",
     [
-        ("bytes=0-", 7489387, 0, 7489387 - 1, None),
-        ("bytes=0-10", 7489387, 0, 10 + 1, None),
-        ("bytes=something", 10, None, None, ValueError),
+        # No end range specified
+        ("bytes=0-", FILE_SIZE, 0, FILE_SIZE - 1, FILE_SIZE, None),
+        # End range == file size
+        (f"bytes=0-{FILE_SIZE}", FILE_SIZE, 0, FILE_SIZE - 1, FILE_SIZE, None),
+        # End range < file size
+        (f"bytes=0-5", FILE_SIZE, 0, 5, 6, None),
+        # End range > file size
+        (f"bytes=0-{FILE_SIZE + 1}", FILE_SIZE, 0, FILE_SIZE - 1, FILE_SIZE, None),
+        # Invalid range
+        ("bytes=something", FILE_SIZE, None, None, None, ValueError),
```

Comment:
What happens in a multipart range request? The range is calculated as `end - start`, but the size of `Range: bytes=0-50, 100-150` is 100, not 150